build_library/prod_image_util.sh

```diff
@@ -76,7 +77,7 @@ create_prod_image() {
 L+  /etc/ld.so.conf     -   -   -   -   ../usr/lib/ld.so.conf
 EOF
 
-  # Only try to disable rw on /usr if there is a /usr partition 
+  # Only try to disable rw on /usr if there is a /usr partition
```

Comment:
I don't follow the concern, you would basically be copying kernel and signing just as finish_image already does, only difference is adding the verity and dd commands to finish_image and removing from here.